home-work-app/src/app/redux/actions/setUpdateStatus.ts

```diff
@@ -0,0 +1,8 @@
+import { IMovieActions } from '../../interfaces/IMovieActions';
+
+export const setUpdateStatus: IMovieActions.setStatus = (isUpdated: boolean): any => (dispatch: any): void => {
+    dispatch({
```

Comment:
This action creator should not `dispatch` action inside, it just should return an action object.
"Object shorthand" form of `mapDispatchToProps` will pass all action creators into `dispatch` automatically.
[https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object](url)